src/adhocracy_core/adhocracy_core/auditing/__init__.py

```diff
@@ -0,0 +1,116 @@
+"""Auditlog of events stored in a ZODB database."""
```

Comment:
may more informative:

    Log which user modifies resources in additional 'audit' database.